### PR TITLE
Performance plot fix

### PR DESCRIPTION
**Pull request by danich1**
This quick PR is designed to update the discriminator model figures and change some code based on the comment in the [previous pr](https://github.com/greenelab/snorkeling/pull/82). Lastly, I appended a new function that will be used in the next pr. 

Minimal code review needed for this one.

### CODE REVIEW

File: figures/label_sampling_experiment/plot_scripts/label_sampling_disc_figure_generator.py [hunk 1]
```diff
@@ -216,10 +219,10 @@ def plot_performance_graph(
     }
 
     color_map = {
-        "DaG": color_names["turquoise"]
-        "CtD": color_names["orange"]
-        "CbG": color_names["purple"]
-        "GiG": color_names["pink"]
+        "DaG": color_names["turquoise"],
```
Comment (ajlee21) [inline figures/label_sampling_experiment/plot_scripts/label_sampling_disc_figure_generator.py:222]: nice!

File: figures/label_sampling_experiment/plot_scripts/label_sampling_gen_figure_generator.py [hunk 2]
```diff
@@ -75,6 +76,8 @@ def plot_performance_graph(
 
             # remove x axis labels
             axes[row_ind][col_ind].set_xlabel('')
+            if metric == "AUROC":
+                axes[row_ind][col_ind].set_ylim([0.5, 1])
```
Comment (ajlee21) [inline figures/label_sampling_experiment/plot_scripts/label_sampling_gen_figure_generator.py:80]: Looks like you still see your same trends, but some are a bit more flat..I don't think any interpretations changed though?

File: modules/utils/notebook_utils/dataframe_helper.py [hunk 3]
```diff
@@ -201,3 +201,37 @@ def generate_results_df(grid_results, curated_labels, pos_label=1):
             model_param_per[param] = [auc(recall, precision), auc(fpr, tpr)]
         performance_dict[lf_sample] = max(model_param_per.items(), key=operator.itemgetter(1))[1]
     return pd.DataFrame.from_dict(performance_dict, orient="index")
+
+def embed_word_to_index(cand, word_dict):
+    return [word_dict[word] if word in word_dict else 1 for word in cand]
+
+def generate_embedded_df(candidates, word_dict, max_length=83):
```
Comment (ajlee21) [inline modules/utils/notebook_utils/dataframe_helper.py:208]: What is this going to be used for, just curious?

**Review by ajlee21 — APPROVED** [review]

File: figures/label_sampling_experiment/plot_scripts/label_sampling_gen_figure_generator.py (hunk 2 above)
Comment (danich1) [reply figures/label_sampling_experiment/plot_scripts/label_sampling_gen_figure_generator.py:80]: Yeah the change on the y-axis makes the trend be more flat, but the interpretation remains the same.

File: modules/utils/notebook_utils/dataframe_helper.py (hunk 3 above)
Comment (danich1) [reply modules/utils/notebook_utils/dataframe_helper.py:208]: So this script update should have been in the next PR, but anyway this code will be used for generating word vectors and encoding sentences into a vector space the discriminator model will use to make predictions.

**Pull request merged**